src/com/rct/homework/lesson2/task07/validator/NoDigitValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.rct.homework.lesson2.task07.validator;
+
+import com.rct.homework.lesson2.task07.localisation.MessageStrings;
+
+/**
+ * Validating strings for containing no numbers.
+ *
+ * @author Oleg Baslak
+ * @version 1.0
+ * @since 08.10.2016
+ */
+public class NoDigitValidator extends Validator {
+
+    //  regex means "no numbers in string"
+    private String symbolsToFind = "\\D*$";
```

Comment:
I have tested it on http://myregexp.com/ on string asdfsd3asdasd and it matches everything after 3. Are you sure it works?